internal/monitor/controller.go

```diff
@@ -249,8 +276,15 @@ func (cm *PodMonitorType) controllerCleanupPod(pod *v1.Pod, node *v1.Node, reaso
 
 	// Call the driver to validate the volumes are not in use
 	if cm.CSIExtensionsPresent && CSIApi.Connected() {
-		log.WithFields(fields).Infof("Validating host connectivity for node %s volumes %v", node.ObjectMeta.Name, volIDs)
+		log.WithFields(fields).Infof("Checking host connectivity for node %s and iosInprogress for volumes %v", node.ObjectMeta.Name, volIDs)
 		connected, iosInProgress, err := cm.callValidateVolumeHostConnectivity(node, volIDs, true)
+		log.WithFields(fields).Infof("Validating host connectivity for node %s volumes %v connected %t iosInProgress %t", node.ObjectMeta.Name, volIDs, connected, iosInProgress)
```

Comment:
suggestion: add some punctuation in the log to make it a bit easier to visually parse when reading the log message.
```suggestion
		log.WithFields(fields).Infof("Validating host connectivity for node: %s, volumes: %v, connected: %t, iosInProgress: %t", node.ObjectMeta.Name, volIDs, connected, iosInProgress)
```